src/charging-station/Bootstrap.ts

```diff
@@ -197,7 +235,6 @@ export default class Bootstrap {
       ),
     };
     await this.workerImplementation.addElement(workerData);
-    this.numberOfChargingStations++;
```

Comment:
The counting will be wrong if you remove that incrementation:  the internal messaging is not the only way to start/stop CS;